### DISCUSSION TITLE
Sunflorea Kingdom Art

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#1583] [Art]
The Sunflorean Empire stretches across all the ocean and into the mainland regions of Sunflower Land. To keep safe, the royal members of the Sunflorean family spend most of the time habitating Sunflorea island.

This is an art request for 'castle' and 'royal' styled assets for the game.

The Kingdom itself will be an island filled with buildings, towers, castles and other medieval royal themed assets.

<img width="372" alt="Screen Shot 2022-10-31 at 1 41 38 pm" src="https://user-images.githubusercontent.com/11745561/198919946-de909d72-4b2c-4507-ad66-3495a4657665.png">

# Requests

There are a range of ways to contribute to Sunflorean Kingdom and we encourage all types of artists to get involved ❤️ As always we lean on the community imagination and there are no limits to what you can provide. We have given a few examples of 'themed' items if you wish to use them.

## 1. Bumpkins

We will want a range of Bumpkin accessories and Bumpkin NPCs for the Sunflorea Kingdom.

You can either contribute wearable items for your Bumpkin or interesting NPCs for in-game.
![fancy](https://user-images.githubusercontent.com/11745561/198920404-96f40e94-34b2-45f6-adcc-99d778fbfd30.png)

Ideas:

- King Costume + Crown
- Queen Costume + Crown
- Wizard Costume
- Knight Costume

## 2. Music

Do you have some ideas that will fit the theme of a royal kingdom? We would love to hear your talent - more details please reach out to @Rofrtd

## 3. Kingdom Decorations + Rare Items

Designs for 'kingdom' based items and decorations. Flags, swords, paintings and fancy rugs!

When designing a decoration or rare item please provide an example of it placed inside of the game in the Land Expansion area.

## 4. Land Decorations

Do you love making the map beautiful for all of our players? Now is your chance to submit decorations that we can use across Sunflorea island.

Examples:

- Towers
- Castle
- House Decorations
- Flags

# Submissions

- Fits game aesthetic and colours of the beach
- Original pieces of work
- Pixel sizing is 1x1px

_By contributing to the project you agree to the terms and conditions - https://docs.sunflower-land.com/support/terms-of-service. Your art can be used freely by the project and all community members_

When submitting art, please include the following:

- Your farm ID (we will make sure the artists get some goodies 🏆 )
- A minified version of your art piece
- An example of the art used inside of the game or on the Bumpkin

## Timeline

Sunflorean Kingdom will likely become available post-christmas, but we are looking to put teaser NPCs and Bumpkin items sooner than that date! 🚀 

As always, contributors will receive an airdrop of the game item they design (if it is tokenised) and also some goodies!

**Comment by VyroxXx** [comment]
Farm ID: 31562

Castle:
![casteloG](https://user-images.githubusercontent.com/114549921/199355423-7fb586f6-f2b6-4eaa-817f-2844e2694360.gif)

83x105
![CasMédio](https://user-images.githubusercontent.com/114549921/199359178-e29639db-4f04-4b75-88c1-38203879bbff.gif)

35x45
![CastePeq](https://user-images.githubusercontent.com/114549921/199357059-88408888-5ed2-494b-931b-e83a6c26b1b5.gif)

Castle model 2:
![CasGrande](https://user-images.githubusercontent.com/114549921/201813495-57f3e471-8212-43bc-b345-0d0f10eb9fc9.gif)

82x105
![Cas82105](https://user-images.githubusercontent.com/114549921/201813547-1606e653-434f-4e1c-9958-6103013fb2cc.gif)

35x45
![Cas3545](https://user-images.githubusercontent.com/114549921/201813595-86a79200-7b9b-49d7-a97e-4dd9ca796837.gif)

Castle model 3:
![Cnovo](https://user-images.githubusercontent.com/114549921/201823215-ca372fb1-d1ea-47e7-bd61-ee00eb09f7c7.gif)

98x105
![CasteloOK](https://user-images.githubusercontent.com/114549921/202299533-52a5e65a-0d0e-407f-9869-f3e5b0f82393.gif)

42x45
![Castelo4245](https://user-images.githubusercontent.com/114549921/202299575-8d402e47-3715-48af-bf7d-2c6038d8ba3e.gif)

Tower:
![tower](https://user-images.githubusercontent.com/114549921/201818179-ac14ef72-fb90-4efb-89e4-9d5719fc56f8.gif)

46x110
![Tower46110](https://user-images.githubusercontent.com/114549921/201818234-1e9dab6e-20fa-4046-90a0-b51473a9e2b3.gif)

21x50
![tower2150](https://user-images.githubusercontent.com/114549921/201818263-79c4bba4-e3a6-4da4-a48b-60383f230bae.gif)

**Reply by MAISAbr** [reply]
Lindoooooo

**Comment by FarmerKuenster** [comment]
Sunflorea Kingdom building ideas!
crafted and build from all the raw materials found in Sunflorea! 

Sunflorea Castle
(59x74)

![round pointy castle-resized](https://user-images.githubusercontent.com/104322749/199635259-2bc86d01-a5fa-4bc3-891a-5226ff99f904.png)
![round pointy castle-resized-large](https://user-images.githubusercontent.com/104322749/199635278-c8ef5187-e016-43a9-b273-45ab1e66d26e.png)

castle without center tower
(59x62)
![round pointy castle without middle tower](https://user-images.githubusercontent.com/104322749/200044152-8cf7edf2-2588-4de4-9223-65957bb4c010.png)
![round pointy castle without middle tower-large](https://user-images.githubusercontent.com/104322749/200044164-5786d809-4618-47e2-8edc-bda9c91a75ec.png)


kingdom village house
(49x64)
![tower and house](https://user-images.githubusercontent.com/104322749/200044237-de455797-21fd-4d61-8c62-dacc141dfc7f.png)
![tower and house-large](https://user-images.githubusercontent.com/104322749/200044250-7ecfc154-2f2e-4792-bb57-43bab07ac888.png)


stand alone tall tower
(27x68)
![round castle tower](https://user-images.githubusercontent.com/104322749/200045812-90e2171b-bae4-4979-9541-1f7c63c02984.png)
![round castle tower-large](https://user-images.githubusercontent.com/104322749/200045825-00709b74-25e8-4ceb-a572-d144a0cf1101.png)



discord: farmerkunester#1626
land ID: #32391

**Reply by VyroxXx** [reply]
similar to mine with some modifications, with different colors taking the last tower from the center putting a bigger one in the center.

**Reply by FarmerKuenster** [reply]
i'm sorry if you think they are similar, its a common "castle" shape, and I was working on this design before I saw what you have submitted. I think both are unique in their own ways.

**Reply by VyroxXx** [reply]
I do not think so, use your creativity please, It is not a common shape, it is usually square or elliptical.

**Comment by LittleEins** [comment]
### Chess Pieces by Little Eins
Farm ID: 73795
![chess-pieces](https://user-images.githubusercontent.com/104184348/201402232-175ac53e-c15d-44c3-b260-94e1ff87102d.png)
Minified version:
![king](https://user-images.githubusercontent.com/104184348/201402246-a93e03fa-47a4-47fc-8b62-2d31f880c8f4.png)
![queen](https://user-images.githubusercontent.com/104184348/201402258-16dbac7c-82af-411a-a799-e4a93fb77eda.png)
![bishop](https://user-images.githubusercontent.com/104184348/201402266-ee7e2fb1-700a-45b0-b772-0b3b0e3d49dd.png)
![rook](https://user-images.githubusercontent.com/104184348/201402274-c065e0a9-782e-45a1-88b0-121ffadc09da.png)
![knight](https://user-images.githubusercontent.com/104184348/201402291-b478c20a-361c-4119-84c8-d167751120a2.png)
![pawn](https://user-images.githubusercontent.com/104184348/201402306-70586f40-d2ac-4b4e-9160-2d6e86190074.png)

**Reply by digital-farms** [reply]
Perfect!

**Reply by adamhannigan** [reply]
These are amazing! Well done 👏

**Comment by FarmerKuenster** [comment]
Some decoration ideas! 
Fancy Sunflorea Kingdom rug
![sunflorea kingdom rug](https://user-images.githubusercontent.com/104322749/209853869-08e1a4da-7d25-49dc-8138-65688fc67906.png)
![sunflorea kingdom rug-large](https://user-images.githubusercontent.com/104322749/209853993-08bc219c-cc1d-4e92-a34a-a1c4d73905ab.png)

Fancy Sunflorea Kingdom picture frame and stand.
Some ideas to go along with it:
1. The stand anyone can get but some of the picture are really rare and hard to get 
2. Some pictures give a boost of some kind. (Just some incentive to try to collect more)
3. Once you collect 3 pictures then a secret boost or perk is unlocked

![sunflorea kingdom paintings and rug on land](https://user-images.githubusercontent.com/104322749/209853699-16e384c5-8a75-42e4-a190-f3cd495d9b8e.gif)

picture frame and stand empty
![painting frame and stand](https://user-images.githubusercontent.com/104322749/209856132-5fb88a6f-24dc-417b-be33-b2b8880c0e89.png)
![painting frame and stand-large](https://user-images.githubusercontent.com/104322749/209856163-bd9363b6-8f14-4b35-8fb2-9c7ec8aee1d6.png)

painting options:
radish sprout painting
![radish sprout painting](https://user-images.githubusercontent.com/104322749/209855002-65492d5c-a85f-4b6b-a0aa-eb213e854cf9.png)
![radish sprout painting-large](https://user-images.githubusercontent.com/104322749/209855204-6012d43c-5706-418a-8df2-36e761d519b5.png)

Sunflower sprout painting
![sunflower sprout painting](https://user-images.githubusercontent.com/104322749/209855249-bdfc5d8a-6f58-478e-828e-09016db3df0d.png)
![sunflower sprout painting-large](https://user-images.githubusercontent.com/104322749/209855264-8fd666b5-6f28-4eb2-863c-2b6abdb6af24.png)

carrot crop painting
![carrot crop painting](https://user-images.githubusercontent.com/104322749/209856678-393edfde-79d9-4bc4-8f4a-e2b28b880a43.png)
![carrot crop painting-large](https://user-images.githubusercontent.com/104322749/209856706-4a9951ed-8334-453c-a6d8-a128c5ea11d6.png)

magic mushroom painting
![magic mushroom painting](https://user-images.githubusercontent.com/104322749/209856361-4cc15646-acc9-470d-8dbd-e064f1aecb2a.png)
![magic mushroom painting-large](https://user-images.githubusercontent.com/104322749/209856374-fa30ef8a-e7f9-4f4d-b0bd-7ea38ad8453f.png)

star filled sky painting
![star filled sky painting](https://user-images.githubusercontent.com/104322749/209856398-e361bda3-25ea-4fdf-bd79-cdb2c946406b.png)
![star filled sky painting-large](https://user-images.githubusercontent.com/104322749/209856409-32d96bd2-3e67-4b78-b9e5-dea0c2e56473.png)

twin mountains painting
![twin mountains painting](https://user-images.githubusercontent.com/104322749/209856474-777836a5-1ec8-4e67-9fc6-b8fab299d3af.png)
![twin mountains painting-large](https://user-images.githubusercontent.com/104322749/209856486-38ef6edf-9df9-4944-8f5f-0ed34b6c7fda.png)

castle tower painting
![castle tower and flowers painting](https://user-images.githubusercontent.com/104322749/209856607-666d8565-372a-436a-ade3-aa5203fdf041.png)
![castle tower and flowers painting-large](https://user-images.githubusercontent.com/104322749/209856615-3f37ba07-a59d-4d3c-93ac-1b16eaf39d69.png)

discord: farmerkunester#1626
land ID: #32391

**Comment by Rylinn** [comment]
Three pillar variations. Just an idea :)
![pillars_example](https://user-images.githubusercontent.com/6690967/219152361-82e336e6-004e-4404-846e-f17ee859623d.jpg)


Pillar 16x32 (16x34 with two more pixel as overlap)
![pillar_16x32_transp34](https://user-images.githubusercontent.com/6690967/219149580-3c9bc966-fa0e-46c3-99f7-88a116f8a757.png)

Pillar broken bottom 16x32
![pillar_broken_16x32](https://user-images.githubusercontent.com/6690967/219150436-2f2253b2-3ee0-4567-be73-29d26be77fd0.png)

Pillar broken top 16x16 (16x17 with one more pixel as overlap)
![pillar_broken_16x17](https://user-images.githubusercontent.com/6690967/219150506-15257bcc-60f6-42bb-84f7-e2a9e9bef03c.png)

Feel free to give me feedback for changes. Thank you for your time

**Comment by Vergelsxtn** [comment]
**Sunflorean Warrior** 🌻🛡️ 

Preview:
without armor design
![Protector-prev2](https://user-images.githubusercontent.com/99605039/223898291-2e2b0e50-2f46-4f67-aedc-96d3fbfe912f.png)
with armor design
![Protector-prev1](https://user-images.githubusercontent.com/99605039/223898294-670c5913-622e-43c0-8592-342d73bcb60e.png)
no helmet
![Protector-prev4](https://user-images.githubusercontent.com/99605039/223898297-c8a47406-e77c-4e2d-a8a3-15909a45fb15.png)

Assets:
![protectors-weapon2](https://user-images.githubusercontent.com/99605039/223898524-5e2dc500-435a-491b-8291-d3d021eb0267.png)
![protectors-weapon1](https://user-images.githubusercontent.com/99605039/223898531-2c977f87-e866-4671-bd1f-36966de26fd8.png)
![protectors-boots](https://user-images.githubusercontent.com/99605039/223898533-d7b76a1a-2ace-4cd7-89b3-4480649bc8b4.png)
![protectors-pants2](https://user-images.githubusercontent.com/99605039/223898536-3cd2c791-81eb-4563-acdc-1cbc408cf4d0.png)
![protectors-pants1](https://user-images.githubusercontent.com/99605039/223898538-d94485ab-44bb-4b52-8e5e-02a31c40647c.png)
![protectors-armor-sleeveless](https://user-images.githubusercontent.com/99605039/223898541-770726e8-c646-4e30-888f-f234a2e24e87.png)
![protectors-armor-with-design](https://user-images.githubusercontent.com/99605039/223898542-20eeebef-e2e7-4285-a99e-626713346796.png)
![protectors-armor-without-design](https://user-images.githubusercontent.com/99605039/223898545-2f37d2fb-07cf-458e-b34e-bd364e03917c.png)
![protectors-armor-nodesign](https://user-images.githubusercontent.com/99605039/223898547-c06c4f6e-4fb4-4537-9f5c-dac3926584cc.png)

![protectors-cape](https://user-images.githubusercontent.com/99605039/223898549-c453c1eb-9f05-476d-ae38-c505fad58664.png)


Discord ID: Vergelsxtn#8727

Farm ID: 63
